### PR TITLE
chore: board-based prioritisation + pipeline regression guards

### PR DESCRIPTION
**Pull request by Hardcoreprawn**
## Summary

Two independent housekeeping items bundled as one chore PR.

### 1. Board-based prioritisation

Switches the project delivery model from a static ROADMAP.md execution table to the live GitHub Project board (`priority:now` / `priority:next` / `priority:backlog` labels). The ROADMAP.md now describes stages and direction; day-to-day ordering lives on the board.

- `docs/ROADMAP.md`: remove static execution-order table, add board URL and label explanation
- `.github/copilot-instructions.md`: update Prioritization section to reference board labels

### 2. Pipeline regression guards (from flakiness investigation)

Adds test coverage for edge cases identified during the pipeline flakiness investigation. All previously untested paths.

- `tests/test_geo.py`: geometry edge cases — collinear, two-point, antimeridian ring coords in `prepare_aoi`
- `tests/test_ingestion.py`: ingestion edge cases
- `tests/test_pipeline.py`: orchestrator helper edge cases, duplicate AOI name handling, `_aggregate_aoi_results` on empty/minimal input
- `tests/fixtures/duplicate_names.kml`: fixture for duplicate-name pipeline smoke scenarios (refs #872)

## Validation

- `python -m pytest tests/test_geo.py tests/test_ingestion.py tests/test_pipeline.py -x -q` → 150 passed

No behaviour changes — tests only (plus docs).

### CODE REVIEW

File: .github/copilot-instructions.md [hunk 1]
```diff
@@ -9,18 +9,21 @@
...
-- Finish Stage 2D (Revenue Enablement) and Stage 2E (Release Safety) before Stage 3 growth features.
-- For every substantial task, identify the primary persona, the job-to-be-done being improved, and the acceptance signal.
-- Prefer thin vertical slices over broad refactors that move several roadmap stages at once.
-- Do not spend engineering time on infrastructure optimisation (T2/T3 split, Container Apps Jobs) until user volume or revenue justifies it.
+Prioritise work in this order:
+
+1. **`priority:now` issues** — these are the current focus. See the live board: [github.com/users/Hardcoreprawn/projects/2](https://github.com/users/Hardcoreprawn/projects/2). Issues labelled `priority:next` form the queue; `priority:backlog` is ordered but unscheduled.
+2. **Stabilise auth, billing, and security** — Stage 2C is complete (2026-05-20). Do not open Stage 3 growth features until auth, billing, and security gates are solid.
+3. **Thin vertical slices** — prefer one stage at a time over broad refactors that touch multiple stages at once.
```
[the remark this answers is not in the document]
Comment (Hardcoreprawn) [reply .github/copilot-instructions.md:16]: Fixed in 43ac868. Updated the Source Of Truth section to say ROADMAP.md holds stages and direction; day-to-day ordering lives on the GitHub Project board. The conflict is resolved."

File: tests/test_ingestion.py [hunk 2]
```diff
@@ -442,3 +442,119 @@ def test_error_message_suggests_next_tier_from_current(self) -> None:
...
+        with pytest.raises(ValueError, match="Duplicate AOI key"):
+            gen.send([])  # resume with empty acquisition results → hits aoi_ref_lookup build
+
+
+class TestEmptyFeatureNames:
+    """Guards against blank names causing all AOIs to share the same claim key."""
+
+    def test_store_claims_batch_empty_name_uses_fallback_key(self) -> None:
+        """Items with empty feature_name get a per-index fallback, not a shared ''."""
+        from treesight.storage.offload import PayloadOffloader
```
[the remark this answers is not in the document]
Comment (Hardcoreprawn) [reply tests/test_ingestion.py:533]: Fixed in 43ac868. `store_claims_batch` now treats empty string the same as `None` — both produce an `item_N` index-based fallback key. Updated test docstring and assertions to verify distinct keys (not just distinct claim_ids).

File: tests/test_pipeline.py [hunk 3]
```diff
@@ -1188,3 +1188,185 @@ def test_poll_order_uses_call_activity_with_retry(self):
...
+        asset_urls, order_meta = _build_order_lookups([])
+        assert asset_urls == {}
+        assert order_meta == {}
+
+    def test_orders_missing_order_id_use_empty_string_key(self):
+        from blueprints.pipeline._helpers import _build_order_lookups
+
+        orders = [{"asset_url": "https://example.com/img.tif"}]  # no order_id
+        asset_urls, _ = _build_order_lookups(orders)
+        assert "" in asset_urls
```
[the remark this answers is not in the document]
Comment (Hardcoreprawn) [reply tests/test_pipeline.py]: Fixed in 43ac868. `_build_order_lookups` now skips orders with no `order_id` instead of inserting them under `""`. Test updated to assert the order is skipped (`asset_urls == {}`).
